__tests__/app.test.js

```diff
@@ -148,6 +148,34 @@ describe("API: /api/reviews", () => {
           });
       });
     });
+
+    describe("GET - CATEGORY: /api/reviews", () => {
+      test("200: responds with array of reviews object filtered by category", () => {
+        const query = "social deduction";
+
+        return request(app)
+          .get(`/api/reviews?category=${query}`)
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toHaveLength(11);
+            reviews.forEach(({ category }) => {
+              expect(category).toBe(query);
+            });
+          });
+      });
+    });
+    describe("CATEGORY - error : /api/reviews", () => {
```

Comment:
What if the category was of the wrong data type for PG to be able to process it?